libraries/libboard/board_n5010/board_n5010.c

```diff
@@ -189,7 +205,9 @@ fpga_result print_bom_info(const fpga_token token)
 		if (mac_addr.ether_addr_octet[3] == 0x2d)
 			serial_name[16] = '0';  //N5013
 		uint32_t number = (mac_addr.ether_addr_octet[4] << 4) + (mac_addr.ether_addr_octet[5] >> 4);
-		printf("%-32s : %s.%04u\n", "Board Serial", serial_name, number);
+		char board_info[FPGA_VAR_BUF_LEN] = { 0 };
+		read_n5014_board_info(token, board_info);
```

Comment:
if (read_n5014_board_info(token, board_info) == FPFG_OK)
   printf("%-32s : %s.%04u (%s)\n", "Board Serial", serial_name, number, board_info)
else
   printf("%-32s : %s.%04u (%s)\n", "Board Serial", serial_name, number, "N/A")

Reply:
OK I will add check, just leaving out the last part in case of failure.